### DISCUSSION TITLE
Sprites on the command line

### DISCUSSION THREAD

**Discussion by orhun** [raphamorim/rio#367] [Ideas]
Hello,

can we habe sprites on the command line pls?

See [0:17](https://youtu.be/o48KzPa42_o?si=flrPpdM4oXjTur9F&t=17) of https://www.youtube.com/watch?v=o48KzPa42_o

**Motivation:** [King Terry](https://en.wikipedia.org/wiki/Terry_A._Davis) would be proud.

I just sponsored your for whole 2 (two) american dollers so hoping that my request won't go over your [HEAD](https://stackoverflow.com/questions/2304087/what-is-head-in-git).

Danke so much,
:*

**Comment by tranzystorekk** [comment]
🙏
